### DISCUSSION TITLE
Manager / and /nsm full

### DISCUSSION THREAD

**Discussion by stondino00** [Security-Onion-Solutions/securityonion#2720] [Unsupported Versions]
I have a manager that I configured as a manager and sensor as well.

All has been fine but within the last week my / and /nsm have filled up to 100%.  I have the stock partitions configured from the install iso.  Is there a cleanup script I can run to remove old items and get the manager back up and running?  Why did the system fill up like that?

Filesystem               Size  Used Avail Use% Mounted on

/dev/mapper/system-root  106G  104G  1.6G  99% /

/dev/mapper/system-nsm   210G  210G   28K 100% /nsm

**Comment by weslambert** [comment]
We have scripts that should manage the disk space, but depending on the throughput, or other conditions, the disk may have filled up before being able to be purged by the next run of the script(s).

For Elastic data, you can run `so-elastic-clear`, however it may not work as desired with the disk as it is.  Otherwise, you can manually delete PCAP, Zeek logs, etc from /nsm.

You may also want to try using something like `du` or similar to determine what is taking up the most space.

**Reply by stondino00** [reply]
I was able to run so-elastic-clear and that seemed to fix the /nsm issue for now.  How can I clean up old SO logs?  That appears to be taking up a lot of space as well under /.


78G     opt
28G     nsm
24G     var
2.0G    usr
163M    boot
104M    root
102M    home
37M     etc
26M     run
20M     tmp

[sohybridad@SO-Manager /]$ cd opt

[sohybridad@SO-Manager opt]$ sudo du -hsx * | sort -rh | head -10

78G     so
0       containerd


[sohybridad@SO-Manager opt]$ cd so

[sohybridad@SO-Manager so]$ sudo du -hsx * | sort -rh | head -10
77G     log
105M    conf
98M     saltstack
14M     rules
12K     gpg
4.0K    tmp
4.0K    __salt.tmp.ckunx717
0       state
0       repo

**Comment by weslambert** [comment]
After looking at this with the rest of the team, it seems there could be a few things going on.  You'll want to reduce `log_size_limit` (/opt/so/saltstack/loca/pillar/minions/$node.sls)  to 50% of available disk space (in GB) for `/nsm`.   Additionally, you may need to check the size of the Strelka log in `/nsm/strelka/log` , removing it and restarting Strelka if necessary (can get quite large -- we are working on rotation).  Last, you may have quite a few so-sensor-clean processes running.  So, you may need to modify the `so-sensor-clean` script as detailed here: https://github.com/Security-Onion-Solutions/securityonion/commit/944817732b54eb640bab941d1f87da066de1178d in order to prevent it self from running multiple instances, then clean up the existing ones, if necessary (find them with `ps aux | grep sensor-clean | grep -v grep`, then kill the processes, or reboot).
